projects/packages/jetpack-mu-wpcom/.phan/config.php

```diff
@@ -10,4 +10,14 @@
 // Require base config.
 require __DIR__ . '/../../../../.phan/config.base.php';
 
-return make_phan_config( dirname( __DIR__ ) );
+$root = dirname( __DIR__, 4 );
+
+return make_phan_config(
```

Comment:
@candy02058912 curious to what's this for?

Reply:
It's the new static analyzer tool we're using pdWQjU-Jb-p2